internal/pkg/cli/deploy.go

```diff
@@ -547,11 +548,48 @@ func (o *deployOpts) askNames() error {
 		return nil
 	}
 
-	names, err := o.sel.Workloads("Select a service or job in your workspace", "")
+	names, err := o.sel.Workloads("Select one or more services or jobs in your workspace.", "")
 	if err != nil {
 		return fmt.Errorf("select service or job: %w", err)
 	}
-	o.workloadNames = names
+	if len(names) == 1 {
+		o.workloadNames = names
+		return nil
+	}
+
+	// Select workload priority by repeatedly prompting with a multiselect until the list of options is depleted.
+	options := names
+	taggedOptions := make([]string, 0, len(options))
+	currentPriority := 1
+	for len(options) > 0 {
+		deploymentGroupIndex := "next"
+		if currentPriority == 1 {
+			deploymentGroupIndex = "first"
+		}
```

Comment:
Potential alternative variable name
```suggestion
		deploymentGroupOrdinalWord := "next"
		if currentPriority == 1 {
			deploymentGroupOrdinalWord = "first"
		}
```